docs/classes/barcodemarker.html

```diff
@@ -140,7 +140,7 @@ <h3>barcode<wbr>Value</h3>
 					<div class="tsd-signature tsd-kind-icon">barcode<wbr>Value<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">number</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/JamesLMilner/THREEAR/blob/6620e61/src/BarcodeMarker.ts#L22">BarcodeMarker.ts:22</a></li>
```

Comment:
Same as above, won't comment any more as these are all autogenerated but hope that makes sense!

Reply:
Oh right, I didn't check the docs. Good thing you did, and thanks for reviewing in the first place.
I'll change these back quickly.